v03_pipeline/lib/tasks/write_metadata_for_run_test.py

```diff
@@ -37,12 +37,12 @@ def test_write_metadata_for_run_task(self) -> None:
                 json.load(f),
                 {
                     'callsets': [TEST_VCF],
-                    'families_failed_missing_samples': {
+                    'family_samples_failed_missing_samples': {
```

Comment:
Looking at this with fresh eyes, there's probably a case to reformat this to:

```
"failed_family_samples" : {
    "missing_samples":
    "relatedness_check":
    "sex_check":
```

or something